CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.1.2 (2020-07-13)
+
+- Fixed the date pipe to allow for data which is not a `Date` object. [#186](https://github.com/blackbaud/skyux-datetime/pull/186)  (Thanks [@Blackbaud-NickGlyder](https://github.com/Blackbaud-NickGlyder)!)
```

Comment:
Does it mean the same thing to say, "Fixed the date pipe to allow `any` as the value type." ?

Reply:
Ya. To me that reads a little funny though as the `any` value type just means that you can pass whatever value type in that you want. So we aren't really "allowing the `any` type" as that is just allowing all types.

Reply:
Fair point! I looked at the associated PR and was curious if it was worth calling out that way.

Reply:
Would it make sense to make change this to something like `Fixed the date pipe to allow all data types.`?

Reply:
Done. I also updated the date.